docs/getting_started/creating_an_admin.rst

```diff
@@ -203,13 +203,21 @@ http://localhost:8000/admin
    :alt: Sonata Dashboard with Category
    :width: 700px
 
+.. note::
```

Comment:
IMO, since this entire document is based on the premise that `sonata-project/doctrine-orm-admin-bundle` is installed, this note should be placed at the beginning. It could indicate that this is a requirement for the examples exposed here, without referencing any error at all.

Reply:
I am against this change, otherwise we need to add notes for all available persistence layers

Reply:
Following the docs will continue result in an error if you do not add the installation of a persistence layer

Reply:
Then we should add this not in a more general way or add link to all available persistence layers installation docs

Reply:
Isn't this enough ? https://sonata-project.org/bundles/admin/3-x/doc/getting_started/installation.html#download-a-storage-bundle